public/index.html

```diff
@@ -6,17 +6,39 @@
     <link
       rel="stylesheet"
       href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
-    />
+      />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Make an appointment to get your COVID vaccine"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+      content="Make an appointment to get your Massachusetts COVID vaccine"
+      />
+
+    <meta name="keywords"
+          content="massachusetts, mass, ma, covid, covid-19, vaccine, vaccines, vax, search, sign up, zipcode"
+          />
+    <meta name="robots" content="index, follow" />
+    <meta name="language" content="English" />
+    <meta name="title" content="Covid-19 Vaccine Appointments in Massachusetts | MACovidVaccines.com" />
+
+    <!-- Open Graph / Facebook -->
+    <meta name="og:type" content="website" />
+    <meta name="og:url" content="https://www.macovidvaccines.com/" />
+    <meta name="og:title" content="Covid-19 Appointments in Massachusetts | MACovidVaccines.com" />
+    <meta name="og:description" content="Make an appointment to get your COVID vaccine in Massachusetts" />
+    <meta name="og:image" content="https://www.macovidvaccines.com/logo192.png" />
+
+    <!-- Twitter -->
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:url" content="https://www.macovidvaccines.com/" />
+    <meta name="twitter:title" content="Covid-19 Vaccine Appointments in Massachusetts | MACovidVaccines.com" />
+    <meta name="twitter:description" content="Make an appointment to get your Massachusetts COVID vaccine" />
```

Comment:
Make an appointment to get your COVID vaccine in Massachusetts (for consistency)

Reply:
Fixed